templates/js/.vscode/launch.json

```diff
@@ -0,0 +1,57 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "command": "yarn start:no-browser",
```

Comment:
I left this as yarn but part of me thinks, should direflow take in yarn or npm as a command and use that here